app/pages/incident/EndorsementAuthorityPage.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.incident
+
+import controllers.incident.routes
+import models.{Index, Mode, UserAnswers}
+import pages.QuestionPage
+import pages.sections.incident.IncidentSection
+import play.api.libs.json.JsPath
+import play.api.mvc.Call
+
+case class EndorsementAuthorityPage(index: Index) extends QuestionPage[String] {
+
+  override def path: JsPath = IncidentSection(index).path \ toString
```

Comment:
Do we need a separate `EndorsementSection` or is this part of incidents?

Reply:
Hmmm, so because an incident is one massive loop in the arrivals, technically this can work as endorsements is a sub part of incidents. But we can have an endorsement section as a sub section of incident potentially.

Reply:
That's ok, sounds like you're all on top of it 🙂 I just wasn't sure with it being called 'incidents and endorsements' how it all pieces together